server/src/main/java/com/exacaster/lighter/application/sessions/SessionService.java

```diff
@@ -74,7 +75,7 @@ public List<Application> fetchByState(ApplicationState state, SortOrder order, I
     public Optional<Application> fetchOne(String id, boolean liveStatus) {
         return applicationStorage.findApplication(id)
                 .map(app -> {
-                    if (app.getState().isComplete() && liveStatus) {
+                    if (!app.getState().isComplete() && liveStatus) {
```

Comment:
I think there was a bug here as i can not imagine information in DB can be fresher then what we can read from backend